<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSettingsValues</class>
 <widget class="QWidget" name="WdgSettingsValues">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>170</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QFormLayout" name="formLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Value</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtValue"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTextEdit" name="txtDesc">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="cmdEdit">
              <property name="toolTip">
               <string>Save</string>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cmdAdd">
              <property name="toolTip">
               <string>Add new</string>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cmdRemove">
              <property name="toolTip">
               <string>Remove selected</string>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblValues">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdRemove</sender>
   <signal>clicked()</signal>
   <receiver>WdgSettingsValues</receiver>
   <slot>removeValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdEdit</sender>
   <signal>clicked()</signal>
   <receiver>WdgSettingsValues</receiver>
   <slot>saveValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdAdd</sender>
   <signal>clicked()</signal>
   <receiver>WdgSettingsValues</receiver>
   <slot>addValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblValues</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>WdgSettingsValues</receiver>
   <slot>selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addValue()</slot>
  <slot>saveValue()</slot>
  <slot>removeValue()</slot>
  <slot>selectionChanged()</slot>
 </slots>
</ui>
